_posts/databases/mysql/2000-01-01-mysql-8-prerequisites.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: MySQL 8 Prerequisites
+title: Prerequisites Before Upgrading to MySQL 8
```

Comment:
@yannski You probably also want to add a `nav` tag with a shorter title for the left menu